webpack.config.ts

```diff
@@ -1,57 +1,18 @@
 /// <reference types="./source/globals" />
 
 import path from 'path';
-import {readFileSync} from 'fs';
 
```

Comment:
```suggestion
```

Reply:
I think actually built-in modules need to be spaced